lib/compress.js

```diff
@@ -57,7 +57,7 @@ function Compressor(options, false_by_default) {
         dead_code     : !false_by_default,
         drop_console  : false,
         drop_debugger : !false_by_default,
-        ecma          : 5,
+        ecma          : 8,
```

Comment:
I think the compress `ecma` default should remain `5`. 
See rationale: https://github.com/mishoo/UglifyJS2/pull/2157#discussion_r123872481